packages/cli/src/util/paths.ts

```diff
@@ -29,7 +29,7 @@ export function createExamplePath(operation: IHttpOperation): Either.Either<Erro
         }),
       );
     }),
-    Either.map(({ template, values }) => parse(template).expand(values)),
+    Either.map(({ template, values }) => new URI.Template(template).expand(values)),
```

Comment:
We can avoid to create an instance and collect it right away.

```suggestion
    Either.map(({ template, values }) => URI.expand(template, values)),
```

Reply:
Done 👍